ynr/apps/candidates/templates/candidates/_edits_disallowed_message.html

```diff
@@ -1,4 +1,4 @@
 <p>
-Editing of data in {{ site.name }} is now disabled, but if you have a correction,
+Editing of this page has been disabled, but if you have a correction,
```

Comment:
Do you think its worth explaining slightly more about _why_ this might be the case here?